src/core/utils/hooks/useTheme.ts

```diff
@@ -12,42 +12,47 @@ export type ThemeType = 'light' | 'dark' | 'os';
  * Defaults to light theme if none provided or set elsewhere.
  * @param theme Light, dark, or based on OS setting.
  */
-export const useTheme = (theme?: ThemeType): void => {
+export const useTheme = (
+  theme?: ThemeType,
+  owningDocument = document,
```

Comment:
Same deal here, prop can be renamed and JSDoc `@param` can be added after line 13.

Reply:
param added